standards.md

```diff
@@ -1,47 +1,126 @@
+* [Pull Request Description](#pull-request-description)
+* [Commits](#commits)
+* [Docs](#docs)
+* [Tests](#tests)
+* [API Design](#api-design)
+* [Functionality](#functionality)
+* [Content](#content)
+* [Code](#code)
 
-Minimize the number of integration tests written and maximize the unit tests!
-Unit test coverage should increase or stay the same with every PR.
+_See also [the Tekton review process](https://github.com/tektoncd/community/blob/master/process.md#reviews)._
 
-This means that most PRs should include both:
+## Pull request description
 
-1. Tests
-2. Documentation updates
+* Include a link to the issue being addressed, but describe the context for the reviewer
+  * If there is no issue, consider whether there should be one:
+    * New functionality must be designed and approved, may require a TEP
+    * Bugs should be reported in detail
+  * Release notes filled in for user visible changes (bugs + features),
+    or removed if not applicable (refactoring, updating tests)
+  * If the template contains a checklist, it should be checked off
 
-## Commit Messages
+## Commits
 
-All commit messages should follow
-[these best practices](https://chris.beams.io/posts/git-commit/), specifically:
+* Follow [commit messages best practices](https://chris.beams.io/posts/git-commit/):
+  1. Separate subject from body with a blank line
+  2. Limit the subject line to 50 characters
+  3. Capitalize the subject line
+  4. Do not end the subject line with a period
+  5. Use the imperative mood in the subject line
+  6. Wrap the body at 72 characters
+  7. Use the body to explain what and why vs. how. Don't just link to an issue and
+      [aim for 2 paragraphs](https://www.youtube.com/watch?v=PJjmw9TRB7s), e.g.:
+      * What is the problem being solved?
+      * Why is this the best approach?
+      * What other approaches did you consider?
+      * What side effects will this approach have?
+      * What future work remains to be done?
+* Prefer one commit per PR; if there are multiple commits, ensure each is
+  self-contained and makes sense without the context of the others in the PR
 
-- Start with a subject line
-- Contain a body that explains _why_ you're making the change you're making
-- Reference an issue number one exists, closing it if applicable (with text such
-  as
-  ["Fixes #245" or "Closes #111"](https://help.github.com/articles/closing-issues-using-keywords/))
+## Docs
 
-Aim for [2 paragraphs in the body](https://www.youtube.com/watch?v=PJjmw9TRB7s).
-Not sure what to put? Include:
+* Include Markdown doc updates for user visible features
+* Spelling and grammar should be correct
+* Try to make formatting look as good as possible (use preview mode to check)
+* Follow [content](https://github.com/tektoncd/website/blob/master/content/en/doc-con-content.md)
+  and [formatting](https://github.com/tektoncd/website/blob/master/content/en/doc-con-formatting.md) guidelines
+* Should explain thoroughly how the new feature works
 
```

Comment:
```suggestion
+ * Ensure that all links are valid
```

Reply:
It would if we could make sure none of the links on the docs are broken.

Reply:
good call! i think we had some automation that tried to do that at some point but i wouldnt be surprised if we turned it off b/c i think it was kinda flakey :S

but yeah, even manually checking links is something i try to remember to when reviewing :S